.nvmrc

```diff
@@ -1 +1 @@
-10.16.2
+14.5.0
```

Comment:
I'm not sure, some packages still support only `<=12.*`.
Also it could be problematic to make it work with our hosting setup on server.

Reply:
The BE works works it just fine (so far :D ). We can test how it will go with FE part and see - if there will be major problems, we can go down to 12.x

Reply:
👍 ok